examples/longbench/eval_longbench_v2.py

```
@@ -0,0 +1,771 @@
+            enable_block_reuse=False,  # RocketKV doesn't support KV cache reuse
+        )
+
+        if args.rocket_sparse:
+            # Configure RocketKV sparse attention
+            sparse_attention_config = RocketSparseAttentionConfig(
+                window_size=args.window_size,
+                kernel_size=args.kernel_size,
+                prompt_budget=args.token_budget,
+                topr=args.topr,
+            )
+            logger.info(f"Using RocketKV sparse attention")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)